src/cml.js

```diff
@@ -159,9 +182,63 @@ class CML {
       ? ''
       : '![CML watermark](https://raw.githubusercontent.com/iterative/cml/master/assets/watermark.svg)';
 
-    const report = `${userReport}\n\n${watermark}`;
+    let userReport = testReport;
+    try {
+      if (!userReport)
+        userReport = await fs.promises.readFile(markdownFile, 'utf-8');
```

Comment:
Im guessing the pre-commit hook changed this because of the line length?
https://github.com/airbnb/javascript#blocks--braces
```suggestion
      if (!userReport) {
        userReport = await fs.promises.readFile(markdownFile, 'utf-8');
      }
```

Reply:
Yes, it's because of line length. We have this format all over the place.

Reply:
Given that this formatting choice is not exclusive to this pull request, I would address it separately. 😅

Reply:
Resolving in favor of https://github.com/iterative/cml/issues/596#issuecomment-1153170448